cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml

```diff
@@ -117,12 +117,12 @@ data:
     }
   {{- end }}
 
-  {{- if and (not .Values.agent) (not .Values.cloudAgent) (not (eq (include "aggregator.deployMethod" .) "disabled")) }}
+  {{- if and (not .Values.agent) (not .Values.cloudAgent) (not (eq (include "aggregator.deployMethod" .) "disabled")) (not .Values.kubecostAggregator.enabled) }}
```

Comment:
While this is correct, I think that `.Values.kubecostAggregator.enabled` is somewhat of a deprecated config. It was primarily used in Kubecost v1.x.

Now, I believe the primary way to configure Aggregator is via `.Values.kubecostAggregator.deployMethod`. More details here:

https://github.com/kubecost/cost-analyzer-helm-chart/blob/bb92e85ffd303fa0063f687728f3931cbf07cde8/cost-analyzer/values.yaml#L2370-L2382

Reply:
@thomasvn thanks for the feedback. if it's deprecated, do you see harm leaving this as is? someone could still technically enable/disable it, which could trigger the issue.

Reply:
FWIW, I haven't used deployMethod yet. I failed to read the comments closer.

we also have an option for agentOnly
which is probably preferable to deployMethod=false  because it kills grafana/frontend/forecasting/aggregator 
?
https://github.com/kubecost/cost-analyzer-helm-chart/blob/develop/cost-analyzer/values.yaml#L3012

These if strings are getting a little long, but I don't have a strong opinion on the right method.